docs/UserGuide.md

```diff
@@ -168,12 +206,169 @@ The keyword refers to the status, role or company that the user intends to look
+
+### Display a list of todo internship applications :`list_todo`
+
+Displays a list of todo applications (todo internship application)
+
+Format: `list_todo`
+
+Examples:
+
+* `list_todo` shows all the todo applications that the user has recorded.
+* If there are no todo applications for at the moment, `No todo at the moment` will be shown.
+
+### Add a todo application: `add_todo`
+
+Adds a todo internship application to the todo list
+
+Format: ` add_todo n/COMPANY_NAME j/JOB_TITLE by/DEADLINE`
+- `DEADLINE` should be in the format yyyy-mm-dd.
+- `DEADLINE` should not be earlier than the date when the todo application is created.
+
+Examples:
+* `add n/Facebook j/Product Manager by/2023-06-07` adds a todo application for the Product Manager role at Facebook. The internship should be applied by 7 June 2023.
+* `add n/LinkedIn j/Software Engineer by/2023-10-04` adds a todo application for the Software Engineer role at LinkedIn.The internship should be applied by 4 October 2023.
+
+### Edit todo application deadline : `edit_deadline`
+
+Edits the deadline of the specified todo from current available todo list
+
+Format: `edit_deadline INDEX by/DEADLINE`
+- Edits the deadline of the specified `INDEX` to the specified `DEADLINE`.
+- The index refers to the index number shown in the displayed todo list.
+- The index must be a positive integer 1, 2, 3, …​
+- `DEADLINE` should be in the format yyyy-mm-dd.
+- `DEADLINE` should not be earlier than the date when the todo application is created.
+
+Examples:
+* `edit_deadline 2 by/2023-07-06` Changes the deadline of the 2nd todo application in the todo list to `2023-07-06` (6 July 2023).
+
+### Edit todo note content : `edit_content`
+
+Edits the note content of the specified todo from current available todo list
+
+Format: `edit_content INDEX c/NOTE_CONTENT`
+- Edits the note content of the specified `INDEX` to the specified `NOTE_CONTENT`.
+- The index refers to the index number shown in the displayed todo list.
+- The index must be a positive integer 1, 2, 3, …​
+- **Note content is an optional field for todo applications**
+- Note content is empty (null) in default.
+
+Examples:
+* `edit_content 2 c/Venue changed` Changes the status of the 2nd todo application in the todo list to `Venue changed`.
+
+### Delete a todo application : `delete_todo`
+
+Deletes the specified todo application from the todo list
+
+Format: `delete_todo INDEX`
+
+* Deletes the todo application at the specified `INDEX`.
+* The index refers to the index number shown in the displayed todo list.
+* The index **must be a positive integer** 1, 2, 3, …​
+* **Note that the action is irreversible**
+
+Examples:
+* `delete_todo 2` Deletes the 2nd todo application in the todo list.
+
+### Clear all todo application entries : `clear_todo`
+
+Clears all todo application entries from the todo applications list
+
+Format: `clear_todo`
+**Note that the action is irreversible**
+
+### Display list of short note :`list_note`
```

Comment:
I think it's better to have a space after ":" and before `list_note` make the whole document more unified